### ISSUE TITLE
Can I use SRT to transfer only binary files and how can I get the fastest speed?

### ISSUE THREAD

**Issue by seamanm** [Haivision/srt#2099]
Can I use SRT to transfer only binary files and how can I get the fastest speed?

[root@vultr srt-1.4.3]# ./recvfile 127.0.0.1 9999 test1.bin data/test1.bin
08:08:31.102665/recvfile D:SRT.sm: generateSocketID: : @4389041
08:08:31.103854/recvfile.N:SRT.cn: @4389041:Connection established to: 127.0.0.1:9999
speed = 183.918Mbits/sec
loss = 9330pkt (52%)
[root@vultr srt-1.4.3]# ./recvfile 127.0.0.1 9999 test1.bin data/test1.bin
08:08:33.421860/recvfile D:SRT.sm: generateSocketID: : @1064618309
08:08:33.422830/recvfile.N:SRT.cn: @1064618309:Connection established to: 127.0.0.1:9999
speed = 370.454Mbits/sec
loss = 3742pkt (21%)
[root@vultr srt-1.4.3]# ./recvfile 127.0.0.1 9999 test1.bin data/test1.bin
08:08:35.209662/recvfile D:SRT.sm: generateSocketID: : @422736432
08:08:35.210772/recvfile.N:SRT.cn: @422736432:Connection established to: 127.0.0.1:9999
speed = 22.9012Mbits/sec
loss = 0pkt (0%)
[root@vultr srt-1.4.3]# ./recvfile 127.0.0.1 9999 test1.bin data/test1.bin
08:08:36.789274/recvfile D:SRT.sm: generateSocketID: : @816794742
08:08:36.790796/recvfile.N:SRT.cn: @816794742:Connection established to: 127.0.0.1:9999
speed = 8.90987Mbits/sec
loss = 0pkt (0%)
[root@vultr srt-1.4.3]# ./recvfile 127.0.0.1 9999 test1.bin data/test1.bin
08:08:38.516761/recvfile D:SRT.sm: generateSocketID: : @402581858
08:08:38.518194/recvfile.N:SRT.cn: @402581858:Connection established to: 127.0.0.1:9999
speed = 9.63709Mbits/sec
loss = 0pkt (0%)
[root@vultr srt-1.4.3]# ./recvfile 127.0.0.1 9999 test1.bin data/test1.bin
08:08:39.892553/recvfile D:SRT.sm: generateSocketID: : @370661619
08:08:39.893718/recvfile.N:SRT.cn: @370661619:Connection established to: 127.0.0.1:9999
speed = 27.8721Mbits/sec
loss = 0pkt (0%)
[root@vultr srt-1.4.3]# ./recvfile 127.0.0.1 9999 test1.bin data/test1.bin
08:08:41.687809/recvfile D:SRT.sm: generateSocketID: : @283335293
08:08:41.688820/recvfile.N:SRT.cn: @283335293:Connection established to: 127.0.0.1:9999
speed = 175.266Mbits/sec
loss = 9867pkt (50%)
[root@vultr srt-1.4.3]# ./recvfile 127.0.0.1 9999 test1.bin data/test1.bin
08:08:44.004384/recvfile D:SRT.sm: generateSocketID: : @895556966
08:08:44.005573/recvfile.N:SRT.cn: @895556966:Connection established to: 127.0.0.1:9999
speed = 302.501Mbits/sec
loss = 7428pkt (41%)
test send and receive at localhost,display loss = 7428pkt (41%),i think local Shouldn't have this

**Comment by maxsharabayko** [comment]
Hi @seamanm 
Yes, SRT can be used to transfer files.
- There is an example `srt-file-transmit` application that performs a congestion-controlled transfer of a single file using [SRT buffer mode](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00#section-4.2.2).
- Another testing application [srt-xtransmit](https://github.com/maxsharabayko/srt-xtransmit). It uses [SRT in message mode](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00#section-4.2.1) and allows transferring multiple files and folders.

- [`srt-stats-plotting`](https://github.com/mbakholdina/srt-stats-plotting) python scripts can be used to visualize SRT stats and see how the transmission goes.
